schema.sql

```diff
@@ -0,0 +1,67 @@
+-- sources
+CREATE TABLE sources(
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(50),
+    PRIMARY KEY(id)
+);
+-- movies
+CREATE TABLE movies(
+    id INT GENERATED ALWAYS AS IDENTITY,
+    publish_date DATE NOT NULL,
+    archived BOOLEAN,
+    silent BOOLEAN,
+    sources_id INT,
+    PRIMARY KEY(id),
+    CONSTRAINT fk_sources FOREIGN KEY(sources_id) REFERENCES sources(id)
+);
+-- genres
+CREATE TABLE genres(
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(50),
+    PRIMARY KEY(id)
+);
+-- music albums
+CREATE TABLE music_albums(
+    id INT GENERATED ALWAYS AS IDENTITY,
+    publish_date DATE NOT NULL,
+    archived BOOLEAN,
+    on_spotify BOOLEAN,
+    genres_id INT,
+    PRIMARY KEY(id),
+    CONSTRAINT fk_genres FOREIGN KEY(genres_id) REFERENCES genres(id)
+);
+-- authors
+CREATE TABLE authors(
+    id INT GENERATED ALWAYS AS IDENTITY,
+    first_name VARCHAR(50),
+    last_name VARCHAR(50),
+    PRIMARY KEY(id)
+);
+-- games
+CREATE TABLE games(
+    id INT GENERATED ALWAYS AS IDENTITY,
+    publish_date DATE NOT NULL,
+    archived BOOLEAN,
+    multiplayer BOOLEAN,
+    last_played_at DATE NOT NULL,
+    games_id INT,
+    PRIMARY KEY(id),
+    CONSTRAINT fk_authors FOREIGN KEY(games_id) REFERENCES authors(id)
+);
+-- Books
+CREATE TABLE books(
+    id INT GENERATED ALWAYS AS IDENTITY,
+    publish_date DATE NOT NULL,
+    archived BOOLEAN,
+    cover_state VARCHAR(150),
+    publisher VARCHAR(150),
+    book_color VARCHAR(50),
+    games_id INT,
+    PRIMARY KEY(id),
+    CONSTRAINT fk_authors FOREIGN KEY(books_id) REFERENCES authors(id)
+);
```

Comment:
- [ ] Great job working on the database but you have just missed one more table the `Label` table please create it as shown in the UML.

Reply:
We totally forgot about it! Already implemetend! Thanks for the feedback 😄